ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/SandboxOnXRunner.scala

```diff
@@ -121,7 +118,7 @@ object SandboxOnXRunner {
           contractIdFeatures = ExperimentalContractIds.of(
             v1 = ExperimentalContractIds.ContractIdV1Support.NON_SUFFIXED
           ),
-          explicitDisclosure = ExperimentalExplicitDisclosure.of(explicitDisclosureUnsafeEnabled),
+          explicitDisclosure = ExperimentalExplicitDisclosure.of(true),
```

Comment:
Could we make this the default in LedgerFeatures ?

Reply:
the feature will stay in experimental mode (at least in 2.6) in Canton so I wouldn't change the default at the moment. In this PR, I am enabling ED by default on SOX for simplification (because we can since SOX is not going to prod). Or do you have a different reason on why this should be changed?